### ISSUE TITLE
Add documentation to tem_tutorial.ipynb for simSPI/path_config.yaml

### ISSUE THREAD

**Issue by geoffwoollard** [compSPI/simSPI#131]
Giving heads up about paths in `simSPI/path_config.yaml` that must exist to avoid errors for non-existing paths.

```
output_dir: './simSPI/temp_workspace/output/'
```

@ninamiolane I tried to push a new feature branch but got

```
remote: Permission to compSPI/simSPI.git denied to geoffwoollard.
fatal: unable to access 'https://github.com/compSPI/simSPI.git/': The requested URL returned error: 403
```

**Comment by geoffwoollard** [comment]
Did the permissions change in the last months? Should I push from my own fork rather than the comSPI organization's fork?

**Comment by ninamiolane** [comment]
@geoffwoollard yes we have restricted access to the branches of compSPI/simSPI to enforce GitHub forks - as is common in public github repositories.

You have to:
- fork the repo
- push your branch there
- open the PR

**Comment by geoffwoollard** [comment]
While [`simSPI/notebooks/path_config.yaml`](https://github.com/compSPI/simSPI/blob/master/notebooks/path_config.yaml) has 
```
output_dir: './notebooks/'
```

[`simSPI/path_config.yaml`](https://github.com/compSPI/simSPI/blob/master/path_config.yaml) does not
